xbmc/utils/FileUtils.cpp

```diff
@@ -226,3 +227,61 @@ CDateTime CFileUtils::GetModificationDate(const std::string& strFileNameAndPath,
   }
   return dateAdded;
 }
+
+bool CFileUtils::ZebraListAccessCheck(const std::string &filePath)
```

Comment:
The first part of this method's implementation looks very much C-style (NULL terminated array, char pointers etc.). Could we C++1x-ify it a bit to match the rest of our code?